app/views/comments/update.js.erb

```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment) %>").replaceWith("<%= j(render @comment) %>");
```

Comment:
hmm this isn't quite right. You are trying to replace the comment in the form so you must pull the unique id from the comments/_form (which you didn't create), let me know if you have questions about this!